src/store/useTaskStore.ts

```diff
@@ -4,6 +4,15 @@ import { Task } from "../models/Task";
 
 const DEFAULT_TASKS = [new Task("Hello world"), new Task("Hello world2")];
 
+const loadTasksFromLocalStorage = (): ITask[] => {
```

Comment:
Move in utils/localstorage.ts and rename getFromStorage and setToStorage, make it reusable

Reply:
And here you have one issue, when you read from storage it's not an instance of Task, it's same as object literal, so after parse you should call Task constructor(new Task()) to recreate the task model with all methods you have in task class item. Also I suggest to create new static method in Task class to create task from objec, and save all other fields such as id and completed